assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -12,117 +12,117 @@
+# A memory request exceeding the memory limit is ignored, and only the limit size is used.
+# Value less or equal to 0 is interpreted as disabling the limit.
 che.workspace.default_memory_request_mb=200
 
-
-# CPU limit default for each container that has no CPU settings in environment.
-# Can be specified either in floating point cores number, e.g. 0.125 or in K8S format integer millicores e.g. 125m
-# Value less or equal to 0 interpreted as limit disabling.
+# CPU limit for each container that has no CPU settings in its environment.
+# Specify either in floating point cores number, for example, `0.125`,
+# or using the Kubernetes format, integer millicores, for example, `125m`.
+# Value less or equal to 0 is interpreted as disabling the limit.
 che.workspace.default_cpu_limit_cores=-1
 
-# CPU request default for each container that has no CPU settings in environment.
-# if default CPU request is more than the CPU limit, request will be ignored,
-# and only limit will be used.
-# Value less or equal to 0 interpreted as disabling this request.
+# CPU request for each container that has no CPU settings in environment.
+# A CPU request exceeding the CPU limit is ignored, and only limit number is used.
+# Value less or equal to 0 is interpreted as disabling the limit.
 che.workspace.default_cpu_request_cores=-1
 
-# RAM limit and request default for each sidecar that has no RAM settings in Che plugin configuration.
-# Value less or equal to 0 interpreted as limit disabling.
+# RAM limit and request for each sidecar that has no RAM settings in the {prod-short} plug-in configuration.
+# Value less or equal to 0 is interpreted as disabling the limit.
 che.workspace.sidecar.default_memory_limit_mb=128
 che.workspace.sidecar.default_memory_request_mb=64
 
-# CPU limit and request default for each sidecar that has no CPU settings in Che plugin configuration.
-# Can be specified either in floating point cores number, e.g. 0.125 or in K8S format integer millicores e.g. 125m
-# Value less or equal to 0 interpreted as disabling limit.
+# CPU limit and request default for each sidecar that has no CPU settings in the {prod-short} plug-in configuration.
+# Specify either in floating point cores number, for example, `0.125`,
+# or using the Kubernetes format, integer millicores, for example, `125m`.
+# Value less or equal to 0 is interpreted as disabling the limit.
 che.workspace.sidecar.default_cpu_limit_cores=-1
 che.workspace.sidecar.default_cpu_request_cores=-1
 
-# Define image pulling strategy for sidecars.
-# Possible values are: Always, Never, IfNotPresent. Any other value
-# will be interpreted as unspecified policy (Always if :latest tag is specified,
-# or IfNotPresent otherwise.)
+# Defines image-pulling strategy for sidecars.
+# Possible values are: `Always`, `Never`, `IfNotPresent`. Any other value
+# is interpreted as unspecified policy. (`Always` is assumed if the `:latest` tag is specified.
```

Comment:
I might remove the parentheses -- this isn't a sidebar but an important aspect of how `:latest` tags are interpreted.

Reply:
```
# Defines image-pulling strategy for sidecars.
# Possible values are: `Always`, `Never`, `IfNotPresent`. 
# If not set or any other value, `Always` will be assumed for images with `:latest` tag, and
# `IfNotPresent` for all other tags and digests.
```